README.md

```diff
@@ -44,6 +44,7 @@ Federated learning research is booming. We organize the papers by their targetin
 
 ### Blogs
 
+* Flower: A Friendly Federated Learning Framework [[Flower Blog]](https://flower.dev/blog/)
```

Comment:
This "Blogs" section is mainly for the general introduction of federated learning. It is not very suitable to include them here. Could you remove this part? The other part is cool.

Reply:
done